test/EVMHost.cpp

```diff
@@ -209,7 +209,7 @@ evmc::result EVMHost::call(evmc_message const& _message) noexcept
 	{
 		// TODO this is not the right formula
 		// TODO is the nonce incremented on failure, too?
-		Address createAddress(keccak256(
+		h160 createAddress(keccak256(
```

Comment:
The other changes apply `h160` -> `Address`. Why are these the other way around?

Reply:
https://github.com/ethereum/solidity/pull/10447#issuecomment-736596814